SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -79,6 +79,21 @@ uint32 GenerateChecksum(GDK_PROPERTY(Property) * Property, uint32 ParentChecksum
 	Checksum = FCrc::StrCrc32(*Property->GetCPPType(nullptr, 0).ToLower(), Checksum); // Evolve by property type
 	Checksum = FCrc::MemCrc32(&StaticArrayIndex, sizeof(StaticArrayIndex),
 							  Checksum); // Evolve by StaticArrayIndex (to make all unrolled static array elements unique)
+#if ENGINE_MINOR_VERSION >= 27 || ENGINE_MAJOR_VERSION == 5
```

Comment:
```suggestion
#if UE_VERSION_NEWER_THAN(4,27,0)
```
I found a nicer solution for this ^